packages/geoprocessing/src/aws/GeoprocessingHandler.ts

```diff
@@ -46,10 +50,21 @@ const WSS_STAGE = process.env.WSS_STAGE || "";
  *
  * @template T the return type of the geoprocessing function, automatically set from func return type
  * @template G the geometry type of features for the geoprocessing function, automatically set from func feature type
+ * @template P extra parameters to pass to geoprocessing function, automatically set from func parameter type
  */
-export class GeoprocessingHandler<T, G = Polygon | LineString | Point> {
+export class GeoprocessingHandler<
+  T = JSONValue,
```

Comment:
This says if the return type of your geoprocessing function is not set, that it will default to a JSONValue.  This is an improvement over `any`